db/migrate/20160907170059_change_order_import_processed_at_type.rb

```diff
@@ -0,0 +1,12 @@
+# This has no effect in MySQL, but corrects a timezone issue in Oracle.
+class ChangeOrderImportProcessedAtType < ActiveRecord::Migration
+
+  def up
+    change_column :order_imports, :processed_at, :datetime
```

Comment:
Now I'm curious if there are more of these out there, like `notifications.dismissed_at` maybe?